server/src/main/java/io/spine/server/commandbus/CommandAckMonitor.java

```diff
@@ -117,6 +132,7 @@ static Builder newBuilder() {
         private StreamObserver<Ack> delegate;
         private TenantId tenantId;
         private SystemWriteSide systemWriteSide;
+        private ImmutableSet<Command> commands;
```

Comment:
Let's discuss. A `Command` may be posted to the instance A, but its outcome will be received on the instance B (the one responsible for processing the entities with the respective ID).